src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100dvw;
```

Comment:
The unit 'dvw' is not a valid CSS unit. Did you mean 'vw' for viewport width? Using 'dvw' will not apply the intended styling, and the width will not be set as expected. Make sure to use the correct unit to ensure the styles are applied correctly.

Reply:
Fix it 
<img width="950" alt="image" src="https://github.com/user-attachments/assets/a5ae27c9-38d5-43c0-9d32-f167d45c63c7">